src/sinks/util/tcp.rs

```diff
@@ -150,6 +151,7 @@ impl TcpSink {
                 TcpSinkState::Connecting(ref mut connect_future) => match connect_future.poll() {
                     Ok(Async::Ready(socket)) => {
                         debug!(message = "connected");
+                        counter!("sinks.tcp_connections_established", 1);
```

Comment:
do we want to thread through the name of the sink here? If we have multiple tcp sinks this will get merged? We probably want to do the same with http.

Reply:
Covered this in the RFC (#2093), but I think this should be handled with `tracing` span data. Exact implementation is TBD.